android_emulator_cli.sh

```diff
@@ -58,17 +96,14 @@ else
 	echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 fi
 
-# Command line tools used to be here
-FILE1=~/Library/Android/sdk/tools/bin/sdkmanager
-# But now they should reside here
-FILE2=~/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager
-# if [ -f "$FILE1" ]; then
-	# echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-	# echo "Android CLI Tools already installed, moving on."
-	# echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-# el
-
-if [ -f "$FILE2" ]; then
+# Java END
+########################################################
+# Android CLI Tools START
+
+# Command line tools used to have another location but now resides here.
+FILE1=~/Library/Android/sdk/cmdline-tools/latest/bin/sdkmanager
+
+if [ -f "$FILE1" ]; then
 	echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 	echo "Android CLI Tools already installed, moving on."
 	echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
```

Comment:
I assume this is no longer an issue since the changes were made two years ago, so I figured I would just remove the commented out stuff.